jekyll/_includes/snippets/runner/container-runner-prereq.adoc

```diff
@@ -0,0 +1,11 @@
+//[#container-runner-prerequisites]
+//== Container runner prerequisites
+
+To install container runners and run jobs, you will need to have root access, and have the following set up and installed:
+
+* link:https://kubernetes.io/[Kubernetes] 1.12+
+* link:https://helm.sh/[Helm] 3.x
+* A runner xref:runner-faqs#what-is-a-runner-resource-class[resource class token]
```

Comment:
I don't this this is a prerequisite, don't we show them to create this in the "Create namespace and resource class" section?